elements/lisk-transactions/src/10_delegate_transaction.ts

```diff
@@ -31,67 +30,30 @@ export interface DelegateAsset {
 	readonly username: string;
 }
 
-export const delegateAssetFormatSchema = {
+export const delegateRegistrationAssetSchema = {
+	$id: 'lisk/delegate-registration-transaction',
```

Comment:
Not to do in this PR but should I create an issue to hash $id from schemas to avoid having this as the key of the object literal holding the schema cache? (it will work like this but just in case someone uses something different here in the future or in customapps)